opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -56,3 +56,18 @@
     ],
     ...,
 ]
+
+
+@dataclass
+class MetricsHistogramAdvisory:
```

Comment:
This is part of the metrics API right? I think we should move it to `opentelemetry.metrics`

Reply:
Yeah, I can move this after the rest is fine

Reply:
Moved in 5cd650ae40f3b92046d34c8d789a3dfa4d52697b